app/models/timed_transitions/transitioner.rb

```diff
@@ -41,24 +39,20 @@ def success?
 
     private
 
-    def is_dummy?
-      @dummy
-    end
-
     def log_level
-      is_dummy? ? :debug : :info
+      @dummy ? :debug : :info
```

Comment:
**not a blocker** 

you could use something like to allow for both use of `dummy` and `dummy?` transparently.
```
attribute :dummy, :boolean, default: false
```

Reply:
This doesn't work as the class isn't an Active Record model. It should probably not be in `app/models`.